Sources/Build/Triple.swift

```diff
@@ -61,7 +61,7 @@ public struct Triple: Encodable {
 
     public enum ABI: String, Encodable {
         case unknown
-        case android = "androideabi"
```

Comment:
[This raw value was added years back and never used](https://github.com/apple/swift-package-manager/commit/9df5dc8b9#diff-98ac448ab57824ad97ff8ae56c8f50afR57) until my recent native Android pulls, so should be fine to change.